contrib/seeds/README.md

```diff
@@ -14,8 +14,11 @@ data. Run the following commands from the `/contrib/seeds` directory:
 ```
 curl https://bitcoin.sipa.be/seeds.txt.gz | gzip -dc > seeds_main.txt
 curl https://mainnet.achownodes.xyz/seeds.txt.gz | gzip -dc >> seeds_main.txt
+curl https://testnet.achownodes.xyz/seeds.txt.gz | gzip -dc > seeds_test.txt
 curl https://bitcoin.sipa.be/asmap-filled.dat > asmap-filled.dat
 python3 makeseeds.py -a asmap-filled.dat -s seeds_main.txt > nodes_main.txt
 cat nodes_main_manual.txt >> nodes_main.txt
+python3 makeseeds.py -a asmap-filled.dat -s seeds_test.txt > nodes_test.txt
+python3 makeseeds.py -a asmap-filled.dat -s seeds_test.txt -m 30000 > nodes_testnet4.txt
```

Comment:
Hm, either I'm missing something or this would mean that the testnet3 nodes would still be included here? Since you say this isn't really working yet (how I understand this [here](https://github.com/bitcoin/bitcoin/pull/30008#issuecomment-2289394756)) maybe rather put a TODO here instead?

Reply:
I've commented it out and added a todo.